data-processing/scripts/run_pipeline.py

```diff
@@ -522,7 +522,7 @@ def run_commands_for_arxiv_ids(
         )
         arxiv_ids = arxiv_ids[: args.max_papers]
 
-    logging.debug(
+    logging.info(
```

Comment:
I want to see the order information in the future log files, so make this an info log.